shared/api/models/CityModel.ts

```diff
@@ -115,6 +118,13 @@ class CityModel {
     return this._latitude
   }
 
+  get chatPrivacyPolicyUrl(): string | null {
+    if (this._chatPrivacyPolicyUrl === '') {
+      return null
+    }
```

Comment:
- [ ] Remove once https://github.com/digitalfabrik/integreat-cms/pull/3785 is merged and released

Reply:
test and prod are ready: https://cms.integreat-app.de/wp-json/extensions/v3/sites